### PR TITLE
update package json after node updgraded

### PR DESCRIPTION
**Pull request by kateriley1**
PR to update package json to start the demo app now that node version has been updated.

Update: have updated to use dotenv to manage the environment variables. All devs need to add .env file to root of repo with:
NODE_OPTIONS=--openssl-legacy-provider

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,11 +12,11 @@
     "homepage": "https://github.com/communitiesuk/uk-gov-dash-components",
     "main": "build/index.js",
     "scripts": {
-        "start": "webpack-serve --config ./webpack.serve.config.js --open --port 8000",
+        "start": "set NODE_OPTIONS=--openssl-legacy-provider && webpack-serve --config ./webpack.serve.config.js --open --port 8000",
```
Comment (Jordan-Hall) [inline package.json:15]: I would use https://www.npmjs.com/package/dotenv in dev and add it to the package .env file.i think it's react version that makes this required

File: package.json [hunk 2]
```diff
@@ -12,11 +12,11 @@
     "homepage": "https://github.com/communitiesuk/uk-gov-dash-components",
     "main": "build/index.js",
     "scripts": {
-        "start": "webpack-serve --config ./webpack.serve.config.js --open --port 8000",
+        "start": "dotenv -- webpack-serve --config ./webpack.serve.config.js --open --port 8000",
         "start-mac": "export NODE_OPTIONS=--openssl-legacy-provider && webpack-serve --config ./webpack.serve.config.js --open --port 8000",
         "validate-init": "python _validate_init.py",
         "prepublishOnly": "npm run validate-init",
-        "build:js": "webpack --mode production",
+        "build:js": "dotenv -- webpack --mode production",
```
Comment (tom-clarke-dluhc) [inline package.json:19]: What is the "--" for here? Should it be "--webpack"?

**Review by tom-clarke-dluhc — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (Ellie-Brakoniecki) [reply package.json:19]: It seems we do need like this  as we are using dotenv-cli - the purpose of the -- is to separate the options intended for dotenv-cli from the subsequent command we want to run (in this case, webpack)

Comment (tom-clarke-dluhc) [reply package.json:19]: Ah, interesting! Thank you for explaining

**Pull request merged**